app/js/main.js

```diff
@@ -63,6 +63,54 @@ const showError = message => {
 let dragState = false;
 let dragSource = null;
 
+document.addEventListener(TOUCH_START_EVENT, handleTouchStart);
+document.addEventListener(TOUCH_MOVE_EVENT, handleTouchMove);
+document.addEventListener(TOUCH_END_EVENT, handleTouchEnd);
+
+function getTouchTargetElement(e) {
```

Comment:
Best Practice ist es, nicht Funktionen direkt zu deklarieren, weil sie dem global Namespace zugeordnet werden. Besser ist:

`const getTouchTargetElement = (e) => { ... };`

So ist sie nur lokal vorhanden und durch den fn short Syntax steht der gesamte lexikalische Scope (inkl. "this") zur Verfügung.

Nachteil ist hier, dass kein Hoisting mehr geht, aber man kann die const vor der Nutzung deklarieren.